source/basic.tex

```diff
@@ -987,14 +987,32 @@
 \end{codeblock}
 \end{example}
 
+\pnum
+A declaration is \defn{name-independent}
```

Comment:
_name-independent_ should be indexed under "declaration", like this:
![1687793866](https://github.com/cplusplus/draft/assets/21071787/be3507fb-19ea-464d-814e-081e8b3ed61d)